app/Actions/Monitoring/GetMetrics.php

```
@@ -90,8 +90,7 @@ private function metrics(
             ->whereBetween('created_at', [$fromDate->format('Y-m-d H:i:s'), $toDate->format('Y-m-d H:i:s')])
             ->select(
                 [
-                    DB::raw('created_at as date'),
-                    DB::raw('ROUND(AVG(load), 2) as load'),
+                    DB::raw('ROUND(AVG(`load`), 2) as `load`'),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'metrics'. (undefined)

(StaticAccess)